exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,17 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
```

Comment:
You could get away with just an `if...else...` here instead of `if... else if.....` A number can only be either positive or negative.

`else if....` is used to add more options if there are more than 2 choices.
E.g.
```
if (colour === 'blue') {
    console.log("sky");
} else if (colour === 'red') {
    console.log("tomato");
} else if (colour === 'yellow') {
   console.log("sun");
} else {
   console.log("colour not recognised");
}

```

Reply:
@louisechow Very good suggestion, I updated it as the following: 

```
function negativeOrPositive(number) {
  if (number < 0){

    return "negative";

  } else {

    return "positive";

  }
}
```